content/en/data_streams/_index.md

```diff
@@ -98,6 +106,11 @@ By filtering to a single Kafka, RabbitMQ, or Amazon SQS cluster, you can detect
 
 Datadog automatically links the infrastructure powering your services and related logs through [Unified Service Tagging][3], so you can easily localize bottlenecks. Click the **Infra**, **Logs** or **Traces** tabs to further troubleshoot why pathway latency or consumer lag has increased.
 
+### Monitor connector throughput and status
+{{< img src="data_streams/connectors_topology.png" alt="A DSM topology map, showing a connector called 'analytics-sink'. The visualization indicates that the connector has a status of FAILED." style="width:100%;" >}}
+
+Datadog can automatically detect your managed [Confluent Cloud][8] connectors and visualize them in the Data Streams Monitoring topology map. Install and configure the [Confluent Cloud integration][9] to collect information from your Confluent Cloud connectors.
```

Comment:
... to collect information from your Confluent Cloud connectors, including throughput, status, and topic dependencies.